test/CMakeLists.txt

```diff
@@ -0,0 +1,177 @@
+project(mapnik-test)
```

Comment:
```diff
diff --git a/plugins/input/postgis/CMakeLists.txt b/plugins/input/postgis/CMakeLists.txt
index ba3192a98..0b804b761 100644
--- a/plugins/input/postgis/CMakeLists.txt
+++ b/plugins/input/postgis/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.15.0)
+cmake_minimum_required(VERSION 3.19)
 project(input-postgis)
 
 find_package(PostgreSQL REQUIRED)
@@ -9,6 +9,7 @@ add_library(input-postgis SHARED
 )
 add_library(mapnik::plugin::input::postgis ALIAS input-postgis)
 
+target_include_directories(input-postgis PRIVATE ${PostgreSQL_INCLUDE_DIR})
 target_link_libraries(input-postgis PRIVATE
     mapnik::headers
     mapnik::mapnik
```
Missing dependencies

Reply:
See above:
the headers should be included with `target_link_libraries(.... PostgreSQL::PostgreSQL)`

https://github.com/Kitware/CMake/blob/master/Modules/FindPostgreSQL.cmake#L302

Reply:
:+1: